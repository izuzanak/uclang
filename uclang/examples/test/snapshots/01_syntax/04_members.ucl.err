 ---------------------------------------- 
Exception: ERROR: in file: "04_members.ucl" on line: 53
      ("demo.static_private: %d\n" % demo.static_private).print();

Cannot access private variable Demo.static_private
 ---------------------------------------- 
