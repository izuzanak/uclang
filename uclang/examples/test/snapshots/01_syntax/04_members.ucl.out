
=== Access to private members throws an exception ==============================


=== Public members are accessible ==============================================

demo.static_public: 1

=== Static members are shared among all instances of class =====================

demo.static_public: 100

=== Non-static members are unique for each class instance ======================

demo.non_static_public: 1

=== Tutorial end ===============================================================

